### ISSUE TITLE
Can you add this feature?

### ISSUE THREAD

**Issue by zhl111** [zsviczian/obsidian-excalidraw-plugin#721]
When there is too much text in an Excalidraw file and it doesn't fit to split into several files. If it is placed into other files, it will be almost impossible to see it clearly. Wouldn't it be nice to add a feature that allows you to customize a part of an Excalidraw file to display the content separately, so that you can place the required content in different files?

![image](https://user-images.githubusercontent.com/29135984/181458977-0cb5a8a6-ed7f-4792-b401-dcc49c9de037.png)

**Comment by zsviczian** [comment]
The feature I have on my wishlist is to select elements on a drawing (A), have the script create a new drawing (B) with those elements, delete the elements from drawing A and replace them with an embedded version of B. Thus you could break up drawings into more atomic drawings. 

Is this what you have in mind?

**Comment by 7flash** [comment]
Would be nice to have "lazy loading" for files with large content.. separate file into tile-sections which are being fetched only once they are zoomed-in..

**Comment by zsviczian** [comment]
Lazy loading would need to be addressed at the Excalidraw core product level. I doubt there will be much appetite by the Excalidraw team to implement such a feature. Decomposing a large canvas into sub canvases each in its own separate Excalidraw file, and using image embeds to create the "master file" where it all comes together is a more likely solution path. Images are loaded asynchronously, though if you have many images, it can take up to several seconds for all these images to load, which might not be desired.

**Comment by trlee02** [comment]
@zhl111 If you haven't already seen, this feature was implemented. See this [video](https://www.youtube.com/watch?v=yZQoJg2RCKI).
